resources/assets/lib/chat/conversation-view.tsx

```diff
@@ -236,6 +239,24 @@ export default class ConversationView extends React.Component<Props> {
     );
   }
 
+  renderUsers() {
```

Comment:
this thing gets rather long if the channel contains a lot of people like in one of the most recent announcement (35 users).

also I wonder if it's supposed to be visible to everyone in the channel or only for announcers...

Reply:
it's visible to everyone and this design is supposed to be temporary